app/assets/styles/_base.scss

```diff
@@ -431,21 +441,27 @@ a:active{
         z-index: 9999;
         color: $base-color;
         width: 80%;
-        max-width: 25rem;
-
+        max-width: 321px;
```

Comment:
why such a specific value?

Reply:
@danielfdsilva This was to match the width of the search bar. certainly there is a non-explicit way to affect the same.